ts/torch_handler/utils/optimization/ipex_optimization.py

```diff
@@ -0,0 +1,46 @@
+from .optimization import optimization_registry, Optimization
+from ..conf.ipex_config import Conf
+import os
+import torch
+import logging
+import subprocess 
+
+logger = logging.getLogger(__name__)
+
+@optimization_registry
+class IPEXOptimization(Optimization):
+    """The Intel® Extension for PyTorch* (IPEX) Optimization.
+    
+    Args:
+        cfg (Conf): the optimization configuration.
+    """
+    def __init__(self, cfg):
+        super().__init__(cfg)
+        
+        self.dtype = cfg.dtype
+        self.channels_last = cfg.channels_last
+
+    def optimize(self, model):

```

Comment:
@msaroufim, for FP32/BF16 eager, as  @jgong5 has suggested, I've kept  them as runtime optimizations as they are light-weight. For INT8 and JIT that include quantization and tracing, as @HamidShojanazeri has suggested, I'll keep them in a documentation. Are we all aligned on this?